### PR TITLE
Proposal: v9

### PR DESCRIPTION
**Pull request by Page-**

### CODE REVIEW

File: test/sbvr/pilots.js [hunk 1]
```diff
@@ -169,7 +169,7 @@ SELECT NOT EXISTS (
 SELECT NOT EXISTS (
 	SELECT 1
 	FROM "pilot" AS "pilot.0"
-	WHERE "pilot.0"."is experienced" = 1
+	WHERE "pilot.0"."is experienced" = TRUE
```
Comment (thgreasi) [inline test/sbvr/pilots.js:172]: I guess changing to real booleans allows us to do an extra optimization for PG in a follow-up minor PR that will convert this to
```
WHERE "pilot.0"."is experienced"
```
right? Extra 4 bytes sql statement size savings :tada:

File: package.json [hunk 2]
```diff
@@ -16,11 +16,11 @@
   "repository": "https://github.com/balena-io-modules/abstract-sql-compiler.git",
```
Comment (thgreasi) [inline package.json:16]: Should we also bump node and add an engines field?

**Review by thgreasi — COMMENTED** [review]
So far lgtm :+1:

File: test/sbvr/pilots.js (hunk 1 above)
Comment (Page-) [reply test/sbvr/pilots.js:172]: Yeah that should be possible in future

File: package.json [hunk 3]
```diff
@@ -57,6 +57,9 @@
       "test/**/*.js"
     ]
   },
+  "engines": {
+    "node": ">=16.13.0"
+  },
```
Comment (fisehara) [inline package.json:62]: Drop:
```
"prepublish": "require-npm4-to-publish",
...
"require-npm4-to-publish": "^1.0.0",
```


```suggestion
  "engines": {
    "node": ">=16.13.0",
    "npm": ">=8.1.0"
  },
```

**Review by fisehara — COMMENTED** [review]
LGTM + NIT

**Review by fisehara — COMMENTED** [review]
LGTM but please let @thgreasi run through it also.
In addition, I'm not on top of all necessary or pending major changes. But this shouldn't block this major release.

**Review by thgreasi — APPROVED** [review]
:tada:

**Pull request merged**